pmsv2/client.go

```diff
@@ -0,0 +1,293 @@
+package pmsv2
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/xendit/xendit-go"
+)
+
+type Client struct {
+	Opt          *xendit.Option
+	APIRequester xendit.APIRequester
+}
+
+func (c *Client) CreatePaymentMethod(data *CreatePaymentMethodParams) (*xendit.PaymentMethodResponse, *xendit.Error) {
+	return c.CreatePaymentMethodWithContext(context.Background(), data)
+}
+
+func (c *Client) CreatePaymentMethodWithContext(ctx context.Context, data *CreatePaymentMethodParams) (*xendit.PaymentMethodResponse, *xendit.Error) {
+	response := &xendit.PaymentMethodResponse{}
+	header := http.Header{}
+
+	if data.XCallbackURL != "" {
+		header.Add("x-callback-url", data.XCallbackURL)
+	}
+	if data.ServiceID != nil {
+		header.Add("service-id", *data.ServiceID)
+	}
```

Comment:
I think these are for internal-use only 🤔 let's remove also

Reply:
removed